modelina-website/src/components/contexts/PlaygroundContext.tsx

```diff
@@ -94,95 +100,108 @@ export const PlaygroundContextProvider: React.FC<{ children: React.ReactNode; }>
     kotlinPackageName: 'asyncapi.models'
   };
 
-  const [showOptions, setShowOptions] = useState(true);
-  const [showOutputNavigation, setShowOutputNavigation] = useState(true);
+  const [showInputEditor, setShowInputEditor] = useState(true);
+  const [showOptions, setShowOptions] = useState(false);
+  const [showOutputNavigation, setShowOutputNavigation] = useState(false);
   const [config, setConfig] = useState<ModelinaOptions>(defaultConfig);
-  const [input, setInput] = useState(JSON.stringify(defaultAsyncapiDocument, null, 4));
+  const [input, setInput] = useState(
+    JSON.stringify(defaultAsyncapiDocument, null, 4)
+  );
   const [models, setModels] = useState<ModelsGeneratorProps[]>([]);
   const [generatorCode, setGeneratorCode] = useState('');
   const [loaded, setLoaded] = useState({
     editorLoaded: false,
-    hasReceivedCode: false,
+    hasReceivedCode: false
   });
   const [showGeneratorCode, setShowGeneratorCode] = useState(false);
   const [error, setError] = useState(false);
   const [statusCode, setStatusCode] = useState(400);
   const [errorMessage, setErrorMessage] = useState('Bad Request');
   const [isLoaded, setIsLoaded] = useState(false);
   const [hasLoadedQuery, setHasLoadedQuery] = useState(false);
-  const [renderModels, setRenderModels] = React.useState<React.ReactNode | null>(null);
+  const [renderModels, setRenderModels] =
+    React.useState<React.ReactNode | null>(null);
 
-  const contextValue = useMemo(() => ({
-    showOptions,
-    setShowOptions,
-    showOutputNavigation,
-    setShowOutputNavigation,
-    config,
-    setConfig,
-    input,
-    setInput,
-    models,
-    setModels,
-    generatorCode,
-    setGeneratorCode,
-    loaded,
-    setLoaded,
-    showGeneratorCode,
-    setShowGeneratorCode,
-    error,
-    setError,
-    statusCode,
-    setStatusCode,
-    errorMessage,
-    setErrorMessage,
-    isLoaded,
-    setIsLoaded,
-    hasLoadedQuery,
-    setHasLoadedQuery,
-    renderModels,
-    setRenderModels
-  }), [
-    showOptions,
-    setShowOptions,
-    showOutputNavigation,
-    setShowOutputNavigation,
-    config,
-    setConfig,
-    input,
-    setInput,
-    models,
-    setModels,
-    generatorCode,
-    setGeneratorCode,
-    loaded,
-    setLoaded,
-    showGeneratorCode,
-    setShowGeneratorCode,
-    error,
-    setError,
-    statusCode,
-    setStatusCode,
-    errorMessage,
-    setErrorMessage,
-    isLoaded,
-    setIsLoaded,
-    hasLoadedQuery,
-    setHasLoadedQuery,
-    renderModels,
-    setRenderModels,
-  ]);
+  const contextValue = useMemo(
```

Comment:
Please don't make unnecessary formatting changes, revert these changes.

Reply:
It's been part of the `eslint-config-prettier` rule and done automatically by prettier from the parent eslint rules, can we have `npm run lint:fix` as we did in the parent directory?

Reply:
Okay sure, can you please create a separate PR for adding the `npm run lint:fix` command.